general/development/policies/accessibility.md

```diff
@@ -121,7 +123,48 @@ See [the ARIA best practice advice on landmarks](https://www.w3.org/TR/wai-aria-
 
 All pages should have a unique title that describes the current page.
 
-See [the WCAG 2.1 success criteria for web page titles](https://www.w3.org/TR/WCAG21/#page-titled) for further information.
+Some tips for providing a meaningful page title:
+
+- The page title must be accurate and informative.
+- If the page causes a change of context (e.g. a search functionality), it should describe the result or change of context to the user.
+- It should be concise.
+- If possible, it should uniquely identify the page.
+- The most identifying information should come first.
+
+#### Example
+
+Consider that a student in the submission page of an assignment called `Kinetics problem set 1` in the `Physics 101` in the `Moodle University`.
+
+Then a suitable page title for the page would be something like:
+
+<ValidExample>
+
+The most unique identifying information first represented by the activity name and its sub-page, then followed by broader identifiers such as the course name and the site name.
+
+`Kinetics problem set 1: Submit assignment | Physics 101 | Moodle University`
+
+</ValidExample>
+
+<ValidExample>
+
+The most unique identifying information first represented by sub-page's name followed by the activity name that the page belongs to, then followed by broader identifiers such as the course name and the site name.
+
+`Submit assignment | Kinetics problem set 1 | Physics 101 | Moodle University`
+
+</ValidExample>
+
+#### Separating components of a page title
+
+When separating the components of the page tile, please use the `moodle_page:TITLE_SEPARATOR` constant.
```

Comment:
Please provide an example of how you may do so?

Reply:
[Example code](https://github.com/moodle/devdocs/pull/729/files#diff-df8f1f100a26a2d67a6012a3b0fe8884c9f6a095acd0b9e0ba1d817104912c69R168-R180) added